packages/devtools_app/lib/src/timeline/flutter/timeline_service.dart

```diff
@@ -80,17 +68,19 @@ class TimelineService {
 
   Future<void> startTimeline() async {
     if (await serviceManager.connectedApp.isAnyFlutterApp) {
-      timelineController.frameBasedTimeline.data = FrameBasedTimelineData(
+      timelineController.data = TimelineData(
           displayRefreshRate: await serviceManager.getDisplayRefreshRate());
+    } else {
+      timelineController.data = TimelineData();
     }
-    timelineController.fullTimeline.data = FullTimelineData();
 
     await serviceManager.serviceAvailable.future;
     await allowedError(serviceManager.service
         .setVMTimelineFlags(<String>['GC', 'Dart', 'Embedder']));
     await allowedError(serviceManager.service.clearVMTimeline());
 
-    final Timeline timeline = await serviceManager.service.getVMTimeline();
+    final vm_service.Timeline timeline =
```

Comment:
nit: is this type on the lhs useful?

Reply:
removed. Also removed prefix -no longer needed